### PR TITLE
secondary features

### PR DESCRIPTION
**Pull request by sbfnk**
Picks out the new features in #308 relating to the secondary model.

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
+# EpiNow2 1.3.3.9000
```
Comment (seabbs) [inline NEWS.md:1]: This was 1.3.4 in the original PR.

**Review by seabbs — APPROVED** [review]
Aside from the news/version mismatch this is good to go.

File: NEWS.md (hunk 1 above)
Comment (sbfnk) [reply NEWS.md:1]: It was meant as indicating dev version as recommended at https://r-pkgs.org/release.html#post-release to be eventually changed into whichever release version this ends up being.

**Comment by seabbs** [comment]
LGTM - I've updated in line with what was on `main`.

**Pull request merged**